tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -123,12 +123,12 @@
 
     <!-- Integration: MySqlConnector -->
     <!-- Assembly: MySqlConnector -->
-    <!-- Latest package https://www.nuget.org/packages/MySqlConnector/2.1.8 -->
-    <PackageReference Include="MySqlConnector" Version="2.1.8" />
+    <!-- Latest package https://www.nuget.org/packages/MySqlConnector/2.1.9 -->
+    <PackageReference Include="MySqlConnector" Version="2.1.9" />
 
     <!-- Integration: NLog -->
     <!-- Assembly: NLog -->
-    <!-- Latest package https://www.nuget.org/packages/NLog/4.7.15 -->
+    <!-- Latest package https://www.nuget.org/packages/NLog/5.0.0 -->
     <PackageReference Include="NLog" Version="4.7.15" />
```

Comment:
Looks like we need to add support for NLog 5.0

Reply:
Good eye. I can try to squeeze this in this week

Reply:
Cool, thanks! We will need to do both logs injection + direct log submission

Reply:
This is being addressed by https://github.com/DataDog/dd-trace-dotnet/pull/2820